### DISCUSSION TITLE
Multi-arch docker image support (docker save, load, etc.)

### DISCUSSION THREAD

**Discussion by bruceliqin** [docker/buildx#2303] [Q&A]
I am currently building x86_64/aarch64 images individually. I build the image with docker build, then save the image as a tar file with docker save. Is something similar supported through docker buildx? I have seen posts of people having issues with this in 2019/2020. Has there been new support on the buildx side?

**Comment by crazy-max** [comment]
Multi-platform is supported by buildx, see https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images for more info.

**Reply by bruceliqin** [reply]
Yes I am able to build an image and export it as a tar file via --output tar, but this would not allow me to load the image again with docker load. I did some research online and it seems that docker load only supports images saved via docker save. The documentation does not mention any of the above.

**Reply by crazy-max** [reply]
> this would not allow me to load the image again with docker load.

Docker store doesn't support multi-platform images. You need to enable containerd snapshotter storage as explained in our docs: https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images
